src/Sections/Home.js

```diff
@@ -1,11 +1,26 @@
 import React from 'react';
+// import { Button } from '../StyledComponents/Button';
+
 class Home extends React.Component {
-render() {
+   render() {
       return (
-         <div>
-            <h1>Home</h1>
+         <div className="container">
+            {/* <h1>Home</h1>
+            <Button primary onClick={() => { console.log('Test clicked!')}}>Test</Button> */}
+            <div className="row">
+               <div className="col-12 col-md-4">
```

Comment:
If you are using React Bootstrap then you should be using its own Bootstrapped Components, like <Row> <Col> etc, whatever there maybe. Unless this is done intentionally.